### PR TITLE
manpage: rewrite for new services

### PR DESCRIPTION
**Pull request by powersj**
Fixes: #188

### CODE REVIEW

**Comment by powersj** [comment]
You can test this by running: `man ./ubuntu-advantage.1`

File: ubuntu-advantage.1 [hunk 1]
```diff
@@ -1,157 +1,130 @@
...
-Current user is not root
-.TP
-.B
-3
-Invalid or missing token when enabling a service
 .TP
-.B
-4
-The requested service is not supported on the current Ubuntu release
+.B detach
```
Comment (OddBloke) [inline ubuntu-advantage.1]: This is the only command not in its alphabetical place; was this intentional?

File: ubuntu-advantage.1 [hunk 2]
```diff
@@ -1,157 +1,130 @@
...
+also disables all enabled services that can be.
+
+
+.SH SERVICES
 .TP
-.B
-5
-Current kernel is too old to support Snaps (required for the Livepatch service)
+.B "Common Criteria EAL2 Provisioning (cc)"
+Enables and install the common criteria artifacts.
```
Comment (OddBloke) [inline ubuntu-advantage.1]: Common Criteria should be title-case.

File: ubuntu-advantage.1 [hunk 3]
```diff
@@ -1,157 +1,130 @@
...
+
+The artifacts include a configure script, a tarball with additional
+packages, and post install scripts. The artifacts will be installed in
+/usr/lib/common-criteria directory. The evaluated configuration guide
```
Comment (OddBloke) [inline ubuntu-advantage.1]: I'm not sure what evaluated means in "evaluated configuration guide"; is it a term of art in Common Criteria?

File: ubuntu-advantage.1 (hunk 1 above)
Comment (powersj) [reply ubuntu-advantage.1]: not intentional, I can't spell...

File: ubuntu-advantage.1 (hunk 2 above)
Comment (powersj) [reply ubuntu-advantage.1]: +1

File: ubuntu-advantage.1 (hunk 3 above)
Comment (powersj) [reply ubuntu-advantage.1]: I copied this from the existing one and started to re-word, but seems I need to continue to re-word

**Comment by powersj** [comment]
I messed the merge up, so closing this one and reopening another.

**Pull request closed without merging**